code/game/objects/structures/prop.dm

```diff
@@ -361,6 +361,39 @@
 	icon_state = "carpprop"
 	density = TRUE
 
+/obj/structure/prop/mainship/prop_sec
+	name = "Security Officer"
+	desc = "They look rather busy and pays no attention to you."
+	icon = 'icons/Marine/mainship_props.dmi'
+	icon_state = "secprop"
+	resistance_flags = RESIST_ALL
+	density = TRUE
+
+/obj/structure/prop/mainship/prop_so
+	name = "Officer"
+	desc = "They look rather busy and pays no attention to you."
+	icon = 'icons/Marine/mainship_props.dmi'
+	icon_state = "officerprop"
+	resistance_flags = RESIST_ALL
+	density = TRUE
+
+/obj/structure/prop/mainship/prop_tech
+	name = "Technician"
+	desc = "They look rather busy and pays no attention to you."
+	icon = 'icons/Marine/mainship_props.dmi'
+	icon_state = "techieprop"
+	resistance_flags = RESIST_ALL
+	density = TRUE
+
+/obj/structure/prop/mainship/prop_sominf
+	name = "SOM Infantry"
+	desc = "They look rather busy and pays no attention to you."
```

Comment:
```suggestion
	desc = "Standard Sons of Mars infantry with a Volkite weapon on hand. While their face is obscured by the visor, it feels like you should keep going instead of loitering around."
```